examples/sync-simple.rs

```diff
@@ -3,7 +3,7 @@ extern crate bus_queue;
 use bus_queue::sync;
 fn main() {
     // Create a sync channel
-    let (tx, rx) = sync::channel(10);
+    let (mut tx, rx) = sync::channel(10);
```

Comment:
removal of ReffCell in the Waker struct requires that the sync::Publisher broadcast method be mutable, requiring that the sync::Publisher object also be mutable.